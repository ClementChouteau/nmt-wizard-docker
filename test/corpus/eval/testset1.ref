There is also likely to be a divergence between the fortunes of multibillion- dollar companies such as Li & Fung, which saw half-yearly sales as of June 2010 rise by almost 20 per cent, and the small, low-tech Hong Kong firms, many of which are based in the Pearl river delta.
Only this week, Charles Schumer and nine other US senators wrote to Mr Geithner telling him to take a tough line on Beijing's refusal to let the International Monetary Fund publish a report they believe concludes that China manipulates its currency.
“China can radically mess up, for example, if it allows asset bubbles to build or if it fails to stave off political upheaval”.
“The economic agendas of each country have a lot to offer each other,” he told a group of US business leaders on Tuesday.
That said, the institutional trappings for the EU-Chinese relationship seem set a rung or two lower than those for China and the US.
If China does not act and the administration does not respond promptly thereafter, the Congress will act.” G20 heads of government meet in Toronto at the end of next week.
“This alliance of authoritarian regimes could pose the most serious threat to the United States and its democratic allies since the collapse of the Soviet empire,” according to Joshua Kurlantzick, a visiting scholar at the Carnegie Endowment.
Not, as it were, so fast.
In February, prices dropped 0.4 per cent from the month before.
is 52 per cent an accurate figure and if so, does it imply an economy predominately based on private ownership?
The Hu visit, in that respect, will not solve anything – it is just a way station on a much longer journey.
Corporate clients range from Nike to Alcatel.
On the Iraqi government's own figures, China is the largest foreign investor in the country's oil sector.
But the collapse of western finance, and the manifest policy failures which caused it, have given the party the ammunition to promote the “superiority of the socialist system” they once lacked.
“Capital outflows have become very sizeable and now eclipse anything seen in the recent past,” Mr Brooks wrote.
The global economy was more likely to enjoy a reasonable recovery over the next two years, benefiting from recent falls in energy prices and exchange rate movements, the International Monetary Fund said yesterday.
The Russian ban on EU food imports, in retaliation for EU sanctions against Ukraine imposed after the invasion of Crimea, has caused a collapse in the price of milk, which is threatening the livelihoods of farmers in the UK.
Senior government officials, including Premier Wen Jiabao, have expressed concern that the value of the dollar will decline, which would erode China's $2,100bn in foreign exchange reserves, mostly held in US government bonds.
The central bank has raised the reserve ratio four times this year.
“If China does not move quickly to continue reforming its economy, it will face a backlash from other international economic stakeholders.
In this view, Beijing has learnt from its 2008-09 stimulus.
Zha Xiaogang, of the Shanghai Institute for International Studies, has published an “economic wish-list”, which includes a relaxation of US restrictions on exports of sophisticated technology to China.
The Pearl river delta has 60m residents too, but is not so much a country as a huge industrial conurbation, with Hong Kong at its southern edge.
Fuhai Zhang, 29, studied engineering at a German university before EBM Papst recruited him last year.
“Until recently there was quite a bit of speculation that the government might be more accommodative because of the weak economic data,” said Louis Kuijs, an economist with Royal Bank of Scotland. “Now it seems that the government wants to be seen as not going for that stimulus, and that does create some downside risks for growth.”
On Wall Street, wary traders speculate on which countries and securities will be under the greatest selling pressure, doubtless to get ahead of the expected wave of selling.
The important point is what comes next in the process.
Goldman's quiet addition as adviser to Mr Li's big corporate reorganisation in January — one week after the deal was announced —
China's outbound investment is increasingly driven by private companies, which have not been affected as much by the anti-corruption campaign.
More likely, however, it is the central bank's way of warning speculators that the Chinese currency is now a two-way bet after it abandoned in June its dollar peg.
Taken together, all these changes combine to make foreign multinationals easy targets to single out.
. to be honest, I do have a certain feeling of regret,” he said.
Asian investors show a clear preference for the automotive, electronics and infrastructure sectors.
But a long delay was not an option;
According to the IHS numbers, world manufacturing output last year came to $8,638bn (€6,979bn, £5,825bn) or 16.7 per cent of global gross domestic product.
Shanghai has ditched its official economic growth target for 2015, becoming the first major city or province in China to abandon such metrics as government policy shifts towards a focus on growth quality over quantity.
China has built scores of new airports in recent years but barely a quarter of the 200 facilities in operation are making any money, with about 150 surviving on government subsidies, the head of Chinese civil aviation said this month.
However, just because China may be less vulnerable to an external financing crisis does not mean that authorities are out of the woods.
Few think the move will have a dramatic impact on the world economy – specifically on the huge global imbalances seen before the financial crisis, which are threatening to re-appear.
The two top sectors – oil and gas and mining – account for more than half of all deals since 2000 and involved nearly $200bn out of a total $368bn in outbound transactions, according to data from Dealogic.
As they gain more experience, companies' role in defining China's foreign policy agenda will grow and their stature on the avenue of the giants will continue to rise.
Over a dinner of spicy fish hotpot, seaweed soup, stir-fried snow peas, potato strips with chilli and a fiery grain liquor that tastes like Chinese medicine, he pours out his anguish while never directly addressing his wife, who looks at him with barely concealed contempt.
Better economic news also supported the market.
Revaluation will divert income from exporters to households, as it should, but the response will then shift income from households (who provide most of the country's net savings) to large borrowers with access to bank credit.
Steve Case, chairman of Revolution and co-founder of AOL said:
Taiwanese businessmen say the combination of a weaker US dollar and government stimulus spending in China has helped drive a recovery in the technology sector in Taiwan.
But while a similar GDP and CPI spike in 2004 sparked fears of overheating and prompted Beijing to launch macroeconomic and administrative cooling measures, Li Xiaochao, the National Bureau of Statistics spokesman, was yesterday notably nonchalant.
It is joining up with China State Shipbuilding Corporation (CSSC) and China Investment Corporation (CIC).
Outside of banks, savers have one other viable investment option:
It is also piling up fresh debts:
I suppose when you were in Japan, you may have heard such views that the war took place 70 years ago, and the war was something done by the past generation and has little to do with the current generation of the Japanese people. Why does the Chinese side refuse to let it go?
There are not the massive mismatches of say big foreign currency borrowing against far smaller local currency revenues that were a feature of the Asian financial crisis.
But analysts expect these thresholds will gradually be lowered.
Doing so may not even subdue industrial fuel consumption that quickly anyway, if Chinese banks continue lending freely despite recent directives to rein in their activities.
It was not supposed to be like that.
You have to combine that with fiscal policy.
Many foreigners see the US as a lawless nation.
What it should do instead is insist on measures that give the signal that investors will face the consequences of loans going sour.
“We have to compromise on levels of education, skills, English.
The European Union is to send a team of senior policymakers to Beijing next month for talks on climate change, the EU's trade deficit with China and other economic issues, officials in Brussels said yesterday.
Rising anti-Chinese sentiment could make life difficult for African governments, whose mining and infrastructure deals with China are central to their countries’ development.
“These investments will deepen US economic engagement in Africa, fuelling growth that will support broader African prosperity and emerging markets for US businesses,” a White House official said.
That is shorthand for two things.
“It's not just the risk that we might lose an employee, but that [he] takes knowledge with him and uses it for his own interests.”
By 2050, China will account for one-third of global GDP, with the rest of Asia taking the region's total contribution to more than one half, he reckons.
The index hit 49.3 last month, down from an average of 67.8 in the first six months.
In response, Dennis Lockhart, president of the Federal Reserve Bank of Atlanta, asked whether that might all go awry.
Poorer, perhaps;
James Kynge is principal at www.ftchinaconfidential.com, an FT research service
Japan is slipping behind its larger neighbour in areas ranging from activity at its ports to foreign direct investment.
“Banks can normally be managed directly but non-bank institutions can't be managed in the same way, so interest rate adjustments are more important.”
Not to worry, goes the emerging consensus.
The rise of these impressive people will change the investment landscape as well, giving investors an alternative between bureaucratic state-owned enterprises, whose interests are often not aligned with profit-seeking shareholders, and a host of get-rich-quick founders who should never have been allowed to list in the first place.
Japan's economy has been the world's second largest, behind the United States’, for just over four decades but this year it is very likely to be overtaken by China.
“But since Chinese customers are still only familiar with instant coffee, we now need to cultivate their habit of drinking freshly brewed coffee.”
Imports from China, which is the country's most politically sensitive trading partner, rose by nearly 12 per cent. That inflated the US trade gap with China by more than 15 per cent to $22.3bn, the biggest since last October.
consumer inflation hit a three-year high of 6.5 per cent in July.
Most analysts anticipate these measures will spur a recovery, and are expecting stabilisation in the third quarter and a more pronounced upturn in the following one.
The overall trade-weighted value of the renminbi has not changed in the past decade but the Chinese currency has strengthened by 30 per cent relative to the dollar during those 10 years.
China's central bank is considering transforming an agency for bailing out troubled state banks into a holding company along the lines of Singapore's Temasek Holdings.
“We are more than just a gateway to China,” says Mr Arculli.
Chinese investments have been on the rise, making up 27 per cent of all investments by non-Japanese investors in the first three months of this year, according to Recof Data, an M&A research firm.
Food price rises actually quickened in January, breaking into double digit territory (+10.3 per cent) – though that was in the run-up to Chinese new year, a time when food prices tend to go up.
A report by the London-based Centre for Economics and Business Research forecast that China would invest £105bn in the British energy, property and transport sectors by 2025.
Mr Tang also implied that only mainland financial institutions would be able to issue renminbi bonds in Hong Kong, rather than Chinese companies generally
By then, the offshore renminbi market will have gained greater power too, and will be testing the mainland's capital controls ever more forcefully.
Just minutes away from Mao's cave, Zhang Peng, 36, is responding to the modern party's call to consumption.
Making the project even more controversial is the need to lay a 38km underwater pipeline linking the terminal to CLP and ExxonMobil's 2,500MW Black Point power plant.
In Shenzhen, authorities plan to raise its minimum wage by 13 per cent to Rmb1,808 next month.
It wasn't clear how many days off they would get every month. And nobody would tell them how many hours of overtime they would be working or whether they would get the $26 deposit for the uniform back.
“The Achilles heel of this economy – ironically – is the trade sector,” he says.
At the end of a two-day meeting in Turkey the representatives, accounting for 85 per cent of the world's output, expressed confidence in the economic forecast in spite of evidence that global growth is falling short of expectations.
Also, the likely head of the AIIB is Jin Liqun, a widely respected senior official at the ADB, as well as at the Chinese sovereign wealth fund CIC, while an Indian will run the Brics Bank.
Hannibal Smith, of the 1980s TV staple The A-Team, had a signature phrase:
Also, China's trade balance is positive (imports are falling faster than exports) and the country has an embarrassingly large trade surplus.
Under this reading, the build-up of large overseas holdings of renminbi through trade will provide the sort of outside pressure for reform of the financial system that entering the World Trade Organisation gave to the real economy a decade ago.
Guangdong's high-tech exports plummeted 37 per cent year on year in January. In August, they improved 1.3 per cent.
This one sounds quite bold:
As long as the government is credible, the banks will be solvent.
One can hardly assert the first and deny the second.
