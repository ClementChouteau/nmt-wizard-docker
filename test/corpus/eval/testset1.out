Large, multi-billion-dollar businesses like Li & Fung, many of which are based in the Pearl River Delta, are likely to face a different fate, as are small, technologically sophisticated Hong Kong companies, which saw sales grow by nearly 20% in the six months to June 2010.
Just last week, Charles Schumer and nine other US senators wrote to Mr. Geithner demanding a tough stance on Beijing's refusal to allow the International Monetary Fund to issue a report believed to have manipulated the renminbi.
"If, for example, China allows its asset bubble to grow bigger or fails to avoid political instability, then China could be in complete chaos."
"The economic agendas of the two countries are highly complementary and aligned, and there is broad scope for cooperation," he told a group of US business leaders on Tuesday.
Nonetheless, the EU-China discussion mechanism appears to be one or two levels lower than similar mechanisms between China and the US.
If China does not act and the Obama administration does not respond quickly after that, Congress will act." The heads of government of the G20 economies will meet in Toronto, Canada, later next week.
"Cooperation between these authoritarian governments could pose the most serious threat to the US and its democratic allies since the collapse of the Soviet Union," said Joshua Kurlantzick, a visiting scholar at the Carnegie Endowment.
Let's just say that while India can leave China behind, it won't be that fast.
House prices fell 0.4% month-on-month in February.
Is 52% accurate? If so, does it mean that it is an economy based primarily on private ownership?
In this respect, Mr. Hu's visit will not solve any problems - it is just a small stop on the way to a long march.
Corporate clients range from Nike to Alcatel.
China is the largest foreign investor in the oil industry, according to data released by the Iraqi government itself.
But the collapse of western finance, and the apparent policy mistakes that led to it, provided the party with ammunition to promote the once-missing "superiority of the socialist system".
"The scale of capital outflows has become very substantial and now exceeds any time in recent years," Brooks wrote.
The global economy is more likely to achieve a reasonably strong recovery in the next two years, benefiting from recent declines in energy prices and currency movements, the International Monetary Fund said yesterday.
Russia's annexation of Crimea and invasion of Ukraine have prompted EU sanctions, while Russia has retaliated by banning imports of European Union food, which has caused milk prices to collapse and threatened the livelihoods of British dairy farmers.
Senior government officials, including Wen Jiabao, China's prime minister, have expressed concern about the potential depreciation of the dollar, which would erode the value of China's $2,100bn foreign exchange reserves, most of which are held in US Treasuries.
The People's Bank of China has raised the reserve requirement ratio four times this year.
"If China does not act quickly and continue its economic reforms, it will face a backlash from other international economic stakeholders.
The view is that the central government has learnt from the 2008-09 stimulus package.
｟ph_unk＃1：査｠ Xiaogang, of the Shanghai Institute for International Studies, has released a "list of economic aspirations", which includes the US easing restrictions on Chinese high-tech exports.
The Pearl River Delta, where Hong Kong is located, also has a population of 60m, but it is not a country after all, but a vast industrial city circle.
Zhang Fuhai, 29, who studied engineering at a German university, was hired by Mr. Ebiant last year.
"Until recently, there was a lot of speculation that the government might be more relaxed because of weak economic data," said Louis Kuijs, an economist at Royal Bank of Scotland. "It now looks like the government wants to show the outside world that it will not do stimulus, which does pose some downside risks to growth."
On Wall Street, wary traders are speculating on which countries and securities will be under the greatest selling pressure, and there is no doubt they want to rush ahead of the expected sell-off.
The point is, what happens next.
In January, a week after Mr. Li's announcement of a large corporate restructuring, Goldman was quietly involved as an adviser, adding another $45bn to the deal that Goldman advised.
Chinese outbound investment is increasingly being driven by private companies, which are less affected by the anti-corruption campaign than state-owned enterprises.
More likely, however, it is the central bank's warning to speculators that the renminbi is now a two-way bet since it abandoned its peg in June.
In short, all these changes make it easy for foreign multinationals to be targeted in the anti-corruption campaign.
To be honest, I do regret," he said.
Asian investors clearly favor India's auto, electronics and infrastructure sectors.
But long delays are not an option;
Last year, world manufacturing output totaled $8,638bn, or 16.7% of global gross domestic product, according to IHS.
Shanghai, which has abandoned its official growth target for 2015, has become the first big city in China to abandon that measure, as the government shifts from over-focusing on the number of growth in the past to focusing more on the quality of growth.
China has built dozens of new airports in recent years, but the head of China's civil aviation administration said this month that only a quarter of the 200 airports that have been put into operation would be profitable, with about 150 depending on government subsidies.
However, just because China may be more immune to the external financial crisis does not mean that Beijing is out of danger.
Few believe the move will have a dramatic impact on the world economy - especially for the huge global imbalances that preceded the global financial crisis, which are now likely to return.
Oil, gas and mining, two of the biggest sectors, account for more than half of all M&A deals since 2000 and nearly $200bn of outbound investment deals in China, according to Dealogic.
As these companies build up more experience, they will play a bigger role in shaping China's foreign policy agenda and their standing on the "SOE Boulevard" will rise.
Spicy fish hotpot, kelp soup, stir-fried Dutch beans, chili-fried potatoes, a bottle of spicy Chinese-medicinal white wine that makes up the family's dinner, Xiang Lihong poured bitter water on the table but never dared to look at his wife.
Good economic news also supported the market.
The revaluation of the renminbi should shift income from exporters to households, but the response would then shift income from households, the main source of China's net saving, to large institutions with access to loans.
Steve Case, chairman of Revolution and co-founder of AOL, said:
Taiwanese business people say the combination of a weak dollar and Beijing's stimulus spending has helped revive the island's technology sector.
Similar developments in China's GDP and CPI in 2004 prompted fears of overheating, prompting Beijing to take macroeconomic and administrative measures to cool the economy, but Li Xiaochao, a spokesman for the National Bureau of Statistics, was fairly calm yesterday.
The company will form joint ventures with China Shipbuilding and CIC.
In addition to depositing money in banks, the Chinese have another viable investment option:
This has also led to the accumulation of new debt:
When you were in Japan, you might have heard some people say that the war has been 70 years old, and it has nothing to do with the present, it is all a matter of the past, so why is China always clinging on?
Asia, for example, has not experienced a serious imbalance between its huge foreign currency borrowings and much smaller local currency revenues, which was once a feature of the Asian financial crisis.
However, analysts expect these thresholds to gradually fall.
If China's banks continue to lend without restrictions despite recent government directives to rein in lending, fuel price rises may not quickly curb fuel consumption in the industry.
This is a far cry from what people expect.
You must combine it with fiscal policy.
Many foreigners see the US as a lawless country.
What it should do is insist on signaling that investors will face the consequences of non-performing loans.
"We have to lower our standards of education, skills and English.
The EU will send a team of senior policymakers to Beijing next month to discuss climate change, the EU's trade deficit with China and other economic issues, EU officials said yesterday.
Growing anti-China sentiment poses a conundrum for African governments, whose mining and infrastructure deals with China are crucial to their own development.
"These investments will deepen U.S. economic cooperation with Africa, boost economic growth to promote prosperity in Africa's wider region and open up emerging markets for U.S. companies," said a White House official.
There are two layers of meaning in this remark.
"The risk is not just that we may lose an employee, but that [he] takes away knowledge and uses it for personal gain."
He estimates that by 2050, China's share of global GDP will reach one-third, and with other Asian countries, Asia as a whole will contribute more than half of global GDP.
Last month it was 49.3, down from an average of 67.8 in the first six months of the year.
As Dennis Lockhart, president of the Federal Reserve Bank of Atlanta, asked whether this would be a complete loss of control.
Poorer, perhaps;
The writer is editor of China Confidential, a research service owned by the Financial Times (｟ph_ent_uri＃1：www.ftchinaconfidential.com｠)
Japan is increasingly lagging behind China, its larger neighbor, in many areas from ports to foreign direct investment.
"The central bank can directly manage the banking system, but it cannot copy this approach to non-bank financial institutions, so interest rate regulation becomes more important."
Don't worry, that's what the latest consensus is.
The rise of these impressive figures will also change the face of investment, giving investors alternatives to bureaucratic state-owned enterprises, which tend to be at odds with profit-seeking shareholders, and those that should not have gone public.
Japan has been the world's second-largest economy for more than 40 years, behind the US, but this year it is highly likely to be replaced by China.
"But because Chinese consumers are still only familiar with instant coffee, we need to cultivate their habit of using instant coffee."
US imports from China, America's most politically sensitive trading partner, rose almost 12%, boosting the trade deficit with China by more than 15% to $22.3bn - the largest since October.
Consumer price inflation hit a three-year high of 6.5% in July.
Analysts generally believe the measures will stimulate the economic recovery and the overall economic growth rate will stabilize in the third quarter and show a clear upward trend in the fourth quarter.
The value of the renminbi has not changed over the past decade, on a trade-weighted basis, but it has risen 30% against the dollar over the decade.
China's central bank is considering turning Central Huijin, which bailed out ailing state-owned banks, into a holding company, following the example of Singapore's Temasek Holdings.
"We are not just the gateway to China," says Mr. Xia.
China's investment share in Japan is growing steadily, accounting for 27% of all non-Japanese investments in the first three months of this year, according to Recof Data, a merger and acquisition research group.
Food prices did pick up somewhat in January, breaking double-digit levels, rising 10.3% - but this is when food prices tend to rise as the Chinese New Year approaches.
A report from the London-based Center for Economics and Business Research predicts that China will invest GBP 105bn in energy, property and transport in the UK by 2025.
Mr. Tang also suggested that only mainland financial institutions could now issue renminbi bonds in Hong Kong, not all Chinese companies.
By then, the offshore renminbi market will also gain greater influence and will increasingly test China's capital controls.
Just minutes from Mao's cave, Zhang Peng, 36, is responding to the party's call for consumption.
To make the project more controversial, it would need to lay a 38km underwater pipeline to connect the receiving station and China Power to ExxonMobil's 2,500MW Black Point.
In Shenzhen, the local government has announced a 13% increase in the minimum wage to CNY 1,808 from February 1.
They don't know how many days a month they can take off, and no one tells them how long they work overtime, or whether they can get the $26 uniform deposit.
"The irony is that China's economy is just as deadly in trade," he said.
The two-day meeting in Turkey came as participants representing 85% of the world's output expressed confidence in economic forecasts, despite recent evidence that global economic growth had failed to meet expectations.
Moreover, the president of the Asian Infrastructure Investment Bank is likely to be the widely respected Jin Liqun, a former senior official at the Asian Development Bank and a former member of China's sovereign wealth fund, China Investment Corporation, while the IDB will be headed by an Indian.
Hannibal Smith, the protagonist of The A-Team, the popular US television series of the 1980s, has a mantra:
Moreover, China's trade balance is positive (imports are falling more rapidly than exports), and China has an embarrassing huge trade surplus.
According to this theory, holding large amounts of the renminbi through trade overseas would create external pressure on the reform of the financial system, similar to the pressure on the real economy brought about by China's accession to the WTO a decade ago.
Exports of high-tech products in Guangdong plunged 37% in January from a year earlier, compared with a 1.3% rise in August.
It sounds bold:
As long as governments are credible, banks will be solvent.
It is hard to deny the second by acknowledging the first.
